src/components/select/utils/get-next-index-by-key.spec.js

```diff
@@ -5,56 +5,103 @@ describe("getNextIndexByKey", () => {
   const endKey = "End";
   const upKey = "ArrowUp";
   const downKey = "ArrowDown";
+  const pageUpKey = "PageUp";
+  const pageDownKey = "PageDown";
   const aKey = "a";
-  const list = ["a", "b", "c", "d", "e", "f"];
+  const list = ["a", "b", "c", "d", "e", "f", "g", "h"];
+  const lastIndex = list.length - 1;
+
+  const PAGE_SIZE = 4;
```

Comment:
Let's import this from the util file rather than declaring it twice

Reply:
I guess you're referring to the PAGE_SIZE const?

I did consider doing that but decided that it's really a UX decision how big the "page" should be, and didn't want to end up in a position where if some developer decided to change the size to something else, the test would automatically pass without at least some acknowledgement of the fact that the behaviour has changed so the test should be updated too. I realise this goes against DRY but I'd rather the test was explicit about what it wants to test, instead of just saying whatever const is chosen in the util file is automatically fine.

Very happy to change this if you disagree, I just thought I should explain what I was thinking.

Reply:
Yeah I disagree, this is a constant so just need to be defined in the one place, if we don't have confidence that the test is correctly testing the behaviour regardless of what value it is, then the test should be rewritten to gain that confidence. 

We don't need to preempt and code around what future devs may or may not be thinking, let's change it 👍🏻